atd-vzd/migrations/default/1700863342765_set_fk_public_atd_txdot_primaryperson_ems_id/up.sql

```diff
@@ -0,0 +1,5 @@
+alter table "public"."atd_txdot_primaryperson"
+  add constraint "atd_txdot_primaryperson_ems_id_fkey"
+  foreign key ("ems_id")
+  references "public"."ems__incidents"
+  ("id") on update restrict on delete set null;
```

Comment:
There's nothing easy to me about visualizing what the `on update restrict on delete set null` does, but I think you have it set just like 100% perfectly here. I read this as, you can't update a record's ID in the `ems__incidents` table if you have primary persons pointing at that ID and if you delete the record out of `ems__incidents`, you end up setting the FK in the primary persons to null.

I did verify how the EMS ETL works, and I think this is all going to work together, but [please take a look](https://github.com/cityofaustin/atd-vz-data/blob/master/atd-etl/afd_ems_import/ems_incident_upload.py#L265-L277) if you don't mind and make sure you agree with me.